### DISCUSSION TITLE
Focus is stolen from inputs in the control panel

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#4] [Q&A] [answered]
On pages such as `/settings/sections/new` the default Craft functionality is to set focus immediately on the `Name` field. With this plugin enabled that default focus state does not happen.

Chrome v98

**Comment by aaronbushnell** [comment]
Thanks for the submission, @kylecotter! Looks like this is a bug with the kbar library so I submitted [an issue](https://github.com/timc1/kbar/issues/171) with a reproducible test case for them to investigate.

**Comment by aaronbushnell** [comment, marked as answer]
Palette 3.0 no longer uses kbar and has been built to prevent the focus from being stolen in the control panel!
